packages/nocodb-sdk/src/lib/audit/v1.ts

```
@@ -0,0 +1,884 @@
+
+  DATA_IMPORT = 'DATA_IMPORT',
+  DATA_EXPORT = 'DATA_EXPORT',
+  USER_PROFILE_UPDATE = 'USER_PROFILE_UPDATE',
+}
+
+export type BulkAuditV1OperationTypes =
+  | AuditV1OperationTypes.DATA_BULK_INSERT
+  | AuditV1OperationTypes.DATA_BULK_UPDATE
+  | AuditV1OperationTypes.DATA_BULK_DELETE;
+
+export interface UserSigninPayload {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)